RocketControlUnitGUI/src/data.ts

```diff
@@ -0,0 +1,103 @@
+// data.ts
+import PocketBase from 'pocketbase';
+
+const pb = new PocketBase('http://127.0.0.1:8090');
+pb.autoCancellation(false);
+
+let isAuthenticated = false;
+
+const ADMIN_EMAIL = 'jgerbrandt@live.com';
+const ADMIN_PASSWORD = 'avionicsSOAR';
```

Comment:
lol now anyone who looks at github can see our password and username for the database (that launches the rocket), lucky no one will do that.. probably but this should not be here 👍

Reply:
right, sorry. I will change this

Reply:
I changed them into enviromental variables.